src/lib/realtime.js

```diff
@@ -9,30 +9,29 @@ const APPS_DOCTYPE = 'io.cozy.apps'
  * @param {object}
  * @returns {Promise}
  */
-async function initializeRealtime({ getApp, onCreate, onDelete, url, token }) {
-  const realtimeConfig = { token, url }
-
+async function initializeRealtime({ getApp, onCreate, onDelete, cozyClient }) {
```

Comment:
How is this handled for `stack-internal` since it doesn't use cozy-client? We still have this case to handle until we officially ask for cozy-client instance for the CozyBar and make a breaking change for that

Reply:
PR to follow this : https://github.com/cozy/cozy-bar/pull/552

Reply:
No more internal ! https://github.com/cozy/cozy-bar/pull/545/commits/b928d06d87d7976e2cc56c91a7af809e52a9956e